src/components/appNavigation/PushButton.js

```diff
@@ -4,7 +4,7 @@ import CustomButton, { type ButtonProps } from '../common/buttons/CustomButton'
 
 type PushButtonProps = {
   ...ButtonProps,
-  canContinue?: Function,
+  canContinue: Promise<boolean>,
```

Comment:
`canContinue` is a function that returns a Promise<boolean>, it's not the value itself

Reply:
indeed, thanks!